signal-desktop.sh

```diff
@@ -53,11 +53,12 @@ case "${SIGNAL_PASSWORD_STORE}" in
         ;;
 esac
 
+# Warn the user about plaintext password
+# - if the user chose basic (this is the default)
+# - and Signal starts for the first time
 if [[ "${SIGNAL_PASSWORD_STORE}" == "basic" ]]; then
-    if [[ -f "${XDG_CACHE_HOME}"/warning-shown ]]; then
-        rm "${XDG_CACHE_HOME}"/warning-shown || true
-    else
-        report_warning
+    if [[ ! -f "${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}/config/Signal/config.json" ]]; then
```

Comment:
This should be $XDG_CONFIG_DIR ? runtime dir is `/run/user/1000`

Reply:
Yes my bad!!

Reply:
"${XDG_CONFIG_DIR}"/Signal/config.json

Reply:
It's XDG_CONFIG_HOME